docs/src/main/sphinx/release/release-371.md

```diff
@@ -0,0 +1,65 @@
+# Release 371 (16 Feb 2022)
+
+## General
+
+* Add support for using secrets in database backed resource group manager
+  configuration. ({issue}`10996`)
+* Add support for the user group selector rule in database backed resource group
+  manager. ({issue}`10914`)
+* Remove `deprecated.disable-set-properties-security-check-for-create-ddl`
+  configuration toggle. ({issue}`10923`)
+* Improve query performance by removing redundant predicates above table scan.
+  ({issue}`10985`)
+* Prevent time outs for planning of certain queries. ({issue}`10985`)
+* Fix performance regression in internal communication authentication
+  processing. ({issue}`10954`)
+* Fix failure when casting values to `decimal(38, 38)`. ({issue}`10946`)
+* Fix transaction idle timeout enforcement. ({issue}`10923`)
+* Fix removal of all catalog session properties when using session property
+  defaults with transactions. ({issue}`10923`)
+
+## Security
+
+* Invoke correct authorization method, `checkCanCreateTable` taking parameters,
+  when table is created via `CREATE TABLE AS`. ({issue}`10939`)
+
+## ClickHouse connector
+
+* Remove support for Altinity version 20.3. ({issue}`10975`)
+* Fix output of table properties with `SHOW CREATE TABLE` statements.
```

Comment:
What was fixed? What was broken?

Reply:
Table properties didn't appear in `SHOW CREATE TABLE` statement. 
cc: @mosabua

Reply:
Table properties were not there in the output .. now they are..